src/Utils.js

```diff
@@ -113,6 +114,13 @@ function childrenOfNode(node) {
   return childrenToArray(children);
 }
 
+function isTextualNode(node) {
+  return typeof node === 'string' || typeof node === 'number';
+}
+
+export function isReactElementAlike(arg) {
+  return React.isValidElement(arg) || isTextualNode(arg) || Array.isArray(arg);
```

Comment:
:+1: one thing that's still not great... a very common mistake will be passing in a string selector: ie `.contains('.button-class')` in such a way that this won't throw, since text is a valid node.

Not sure what we can do about that at this point though...